docs/css/app.css

```diff
@@ -46,3 +46,23 @@ button.dl {
   text-align: center;
   margin-top: 150px;
 }
+
+/* Restore drop-shadow for the header. */
+header.md-header {
+  -webkit-transition: background-color .25s, color .25s, box-shadow .25s;
+  transition: background-color .25s, color .25s, box-shadow .25s;
+  box-shadow: 0 0 0.2rem rgba(0, 0, 0, .1), 0 0.2rem 0.4rem rgba(0, 0, 0, .2);
+}
```

Comment:
Drop shadows in the middle of the top bar looks weird. I added this custom CSS because tabs were no longer needed. Wanna remove this?

Current:
![image](https://user-images.githubusercontent.com/2387680/84941816-e4a4ea80-b0af-11ea-9937-2041ffa2f057.png)


After:
![image](https://user-images.githubusercontent.com/2387680/84941779-d8209200-b0af-11ea-8373-41649101094a.png)

Reply:
yup that looks great, fixed